docs/src/_getting-started/meltano-at-a-glance.md

```diff
@@ -0,0 +1,78 @@
+
+### What users say
+
+_"For us it's a better day at work when we can use Meltano."_ - Nino Müller, Head of Technology at Substring
+
+
+_"I love Meltano because it’s so pleasant to use with its DevOps and Everything-as-Code style. It is easy to set up, flexible, and integrates with pretty much any orchestrator as well as the data built tool (dbt)"._  - Martin Morset
+
+## Meltano at a glance
+Welcome to your Open Source DataOps Infrastructure to manage and centralize all the data tools in your stack and turn them into your ideal end-to-end data platform. With Meltano, you'll be able to collaboratively build and improve your data platform like a software project; spinning up a service or tool (Airflow, dbt, Great Expectations, Snowflake, etc) and easily configuring, deploying and managing it through a single control plane.
+
+
+### Why companies love to build with Meltano
+
+- **No lock-in**: It's open source and has a strong community, you'll always be free.. (derisk)
+- **It's extensible from day 1**: It's super easy to add a custom connection using the SDKs/EDKs.
+- **Amazing developer experience**: Developers go from start to finish on new data projects, including extraction, loading, transforming & orchestrating data within days.
+- **Small surface area**: Features like "inline transformations" remove unnecessary steps from your data pipelines, and help companies stay compliant to security & GDPR regulations.
+
+### Key Features of Meltano, developers will love
+- **Start simple**: Meltano is pip-installable and comes in a prepackaged docker container, you can have your first ELT pipeline running litterally within minutes.
+- **DataOps out-of-the-box**: Meltano provides tools that make DataOps best practices easy to use in every project.
+- **Integrates with everything**: 300+ natively supported data sources & targets, as well as additional plugins like great expectations or dbt are natively available. 
+- **Easily customizable**: Meltano isn't just extensible, it's built to be extended! SDK & EDK are easy to use, plugins created by others are available on the Hub.
+- **Mature system**: Developed since [2018](https://handbook.meltano.com/timeline), runs in production at large companies like GitLab, and currently powers over a million pipeline runs monthly.
+- **First class ELT tooling built-in**: Extract data from any data source, load into any target, use inline maps to transform on data on the fly, and test the incoming data with great expectations, all in one package. 
+
+### The Quick Introduction
+Waiting to see how Meltano works within 90 secs? We got you covered:
+<iframe width="800" height="632" src="https://www.youtube.com/embed/53WC4kTwbGU" title="From 0 to ELT in 90 seconds with Meltano, tap-gitlab, and target-postgres" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Love the inline videos. I do wish each of these videos had better title cards. 

The only downside to this video is that it is _old_. A point for later would be to re-run since so much has improved about the command line experience.

Reply:
Yes I agree, but I think that's still WIP.